CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.11.0 Release
+- Release Date: August 16, 2018
+- Server Versions Supported: Server v4.0+ is required, Self-Signed SSL Certificates are not supported
+
+### Highlights
+
+#### Searching Archived Channels
+- Added ability to search for archived channels on desktop and mobile clients.
+
+#### Deep Linking
+- 
+
+### Improvements
+- Added profile pop-up to combined system messages.
+- Updated the home screen icon for Android.
```

Comment:
I don't think the screen icon is an improvement, what could be an improvement is Force re-entering SSO auth credentials after logout

Reply:
Improvements on consecutive posts by the same user

Reply:
(Also maybe an improvement from https://github.com/mattermost/mattermost-mobile/pull/1972)
- Added a loading indicator when user info is still loading on LHS